public/source/storage.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * A wrapper around LocalStorage to support environments where LocalStorage is not available.
+ * Stores and retrieves items from LocalStorage if available and uses a non-persistent cache otherwise.
+ */
+var storage = {};
+try {
+    storage.getItem = localStorage.getItem;
+    storage.setItem = localStorage.setItem;
+} catch (e) { /* Ignore Exception, use fallback implementation. */ }
+
+if (!storage) {
```

Comment:
This will not work because `storage` is already an empty `object` (`{}`) which is `true` when converted to a boolean in JavaScript.